ports/espressif/boards/lilygo_tdeck/mpconfigboard.mk

```diff
@@ -0,0 +1,14 @@
+USB_VID = 0x303A
+USB_PID = 0x8161
```

Comment:
Don't just randomly change this. Get a PID assigned from Espressif. The vendor id 0x303A is theirs and they administer it here: https://github.com/espressif/usb-pids

Reply:
Much appreciated. I've been looking for this for the past few days.